app/components/mandatarissen/mandataris-edit-prompt.js

```diff
@@ -59,7 +60,7 @@ export default class MandatenbeheerMandatarisEditPromptComponent extends Compone
       sourceGraph: SOURCE_GRAPH,
     });
 
-    setTimeout(() => this.args.mandataris.reload(), 1000);
+    setTimeout(() => this.router.refresh(), 1000);
```

Comment:
refresh reloads all models up to this point, which is safer. It also means that if we request relations to be included those will be reloaded too